src/components/ClusterSettingsPage/ClusterSettingsPage.module.css

```diff
@@ -0,0 +1,271 @@
+    display: flex;
+    justify-content: space-between;
+  }
+  
+  .UpdateProjectModelButtons {
+    display: flex;
+    justify-content: space-between;
+    
+  }
+  
+  .ModalUpdateForm {
+    display: grid;
+    gap: 2rem;
+  }
+  
+  .ModalUpdateForm .ModalFormInputs {
+    display: grid;
+    gap: 1rem;
+  }
+  
+  .ModalUpdateForm .ModalFormInputs .InputText {
+    border: 2px solid #000000;
+    color: #000;
+  }
+  
+  .ModalUpdateForm .ModalFormInputs .InputText::placeholder {
+    color: #000;
+  }
+  
+  .ModalUpdateForm .ModalFormButtons {
+    display: flex;
+    justify-content: space-between;
+  }
+  
+  .ProjectDeleteModel .ModalChildWrap {
+    min-width: 35rem;
+   
+  }
+  
+  .UpdateForm {
+    display: flex;
+    gap: 1rem;
+    padding: 1rem 1.3rem;
+    flex-direction: column;
+  }
+  
+  .UpdateInputSection {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+  }
+  
+  .ContentSection {
+    display: flex;
+    flex-direction: column;
+    overflow-y: auto;
+  }
+  
+  .ErrorOnWhite {
+    width: 25rem;
+  }
+  .SideBarSection {
+    overflow: hidden;
+    overflow-y: scroll;
+    height: calc(100vh - 5rem);
+  }
+  
+  .SideBarSection::-webkit-scrollbar {
+    display: none;
+    min-width: fit-content;
+  }
+  
+  .Page {
+    margin: 0;
+    background: #f1f1f1;
+  }
+  
+  .MainSection {
+    /* divided into two columns */
+    background-color: #f1f1f1;
+    display: grid;
+    grid-template-columns: 1.3fr 6fr;
+    overflow: hidden;
+  }
+  
+  .MainContentSection {
+    /* divided into two rows */
+    display: grid;
+    grid-template-rows: 80px 4fr;
+    overflow: hidden;
+  }
+  .TopBarSection {
+    background-color: black;
+  }
+  
+  .SideBarSection {
+    -ms-overflow-style: none;
+    scrollbar-width: none;
+  }
+  .ContentSection {
+    display: flex;
+    flex-direction: column;
+    overflow-x: hidden;
+    overflow-y: auto;
+    /* padding-left: 3rem; */
+  }
+  .InnerModalDescription {
+    display: flex;
+    flex-direction: column;
+    gap: 0.4rem;
+  }
+  .DeleteWarning {
+    color: #f00;
+  }
+  .InactiveDelete {
+    background-color: #fff;
+    outline: #ff9595;
+    color: #ff9595;
+  }
+  .InactiveDelete:hover {
+    background-color: #fff;
+    outline: #ff9595;
+    color: #ff9595;
+  }
+  .WarningContainer {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+  }
+  .DeleteSubDescription {
+    font-weight: 500;
+  }
+  
+  .ProjectInstructions {
+    padding: 1rem 1rem;
+    background: #ffffff;
+    border: 1px solid var(--color-gray-light);
+    display: grid;
+    gap: 2rem;
+  }
+  
+  .ProjectButtonRow {
+    display: grid;
+    grid-template-columns: auto 40%;
+    grid-auto-flow: column;
+    align-items: center;
+    column-gap: 1rem;
+  }
+  
+  .SettingsSectionInfo{
+    display: flex;
+    flex-direction: column;
+    gap: 0.4rem;
+  }
+  
+  .SectionButtons {
+    display: flex;
+    justify-content: flex-end;
+    padding-right: 1rem;
```

Comment:
@khalifan-kfan I suggest at a UX level we can not have a call to action with the same color as its background, It loses its essence as a call to action, I suggest we use the default button background color of #f7b21f